public/pages/Transforms/components/Indices/Indices.tsx

```diff
@@ -24,32 +24,32 @@ interface IndicesProps {
 }
 
 export default class Indices extends Component<IndicesProps> {
- constructor(props: IndicesProps) {
-   super(props);
- }
+  constructor(props: IndicesProps) {
```

Comment:
Is the additional space intended for this part of code?

Reply:
I think this may have been an unintended side-effect of cleaning up some comments and foobarring. Looks like it happened in the `render` method as well.